lib/spack/spack/test/cmd/mirror.py

```diff
@@ -154,48 +135,44 @@ def test_exclude_file(mock_packages, tmpdir, config):
     assert not any(spec.satisfies(y) for spec in mirror_specs for y in expected_exclude)
 
 
-def test_mirror_crud(tmp_scope, capsys):
+def test_mirror_crud(mutable_config, capsys):
```

Comment:
This is a good change.

Reply:
I forgot to motivate the change I think, but the Real Reason is that argparse validates --scope values with a static list, so the first times `spack mirror` is executed it's fixed and custom scopes are no longer allowed. The fact that it worked so far is just by chance, randomizing the test order makes it break most of the time.